src/context/ParamsContext.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Params } from "../interfaces";
+const ParamsContext = React.createContext<Params | undefined>(undefined);
+
+export const ParamsContextProvider: React.FC<{ params: Params }> = ({
```

Comment:
Could we get some comments here explaining what the params could be?

Reply:
All the possible values are in the `Params` interface though. It'd be redundant to add them here again.